packages/marketplace/src/components/ui/developer-invite-member-modal/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './developer-invite-member-modal'
```

Comment:
I think this file should change to index.ts for consist entire the project marketplace